### ISSUE TITLE
Some missing items/models error

### ISSUE THREAD

**Issue by Gunzukubo** [Aldrin-John-Olaer-Manalansan/DOTA-2-MOD-Master#73]
![qop_crimson_matriarch_shoulder vmdl_c](https://user-images.githubusercontent.com/114898463/193505901-07e6899f-2a67-47a3-8ad6-548c42d3f4cf.png)
![nightstalker_tail_night vmdl_c](https://user-images.githubusercontent.com/114898463/193505911-bdf492ae-2446-47e2-9d7d-550a9e096b72.png)
![nightstalker_legarmor vmdl_c](https://user-images.githubusercontent.com/114898463/193508757-25b73946-158f-4865-b442-4616cf3f1286.png)

**Comment by epic79** [comment]
Yeah I got those too but they're just notifications and they don't affect anything in-game and those items works correctly
